bin/dev-server.js

```diff
@@ -82,23 +82,6 @@ function watchAll() {
     debounce(rebuildPerf, 700, {leading: true}));
 }
 
-var filesWritten = false;
-
-Promise.resolve().then(function () {
-  if (process.env.CI) {
-    return; // don't bother rebuilding in CI; we already built
-  }
-  return Promise.all([
-    rebuildPouch(),
-    rebuildTestUtils(),
-    rebuildPerf()
-  ]);
-}).then(function () {
-  console.log('Rebuilt PouchDB/test/perf JS bundles');
-  filesWritten = true;
-  checkReady();
-});
-
```

Comment:
I may have just caused a conflict here (?), but happy to help resolve - shouldn't be too complicated.

Reply:
No problem, I've just removed the lines again.